libcontainer/process_linux.go

```diff
@@ -160,6 +161,47 @@ func (p *setnsProcess) start() (retErr error) {
 		case procHooks:
 			// This shouldn't happen.
 			panic("unexpected procHooks in setns")
+		case procSeccomp:
+			// receive seccomp-fd
+			pidfd, _, err := unix.Syscall(unix.SYS_PIDFD_OPEN, uintptr(p.pid()), 0, 0)
+			if err != 0 {
+				return fmt.Errorf("error performing SYS_PIDFD_OPEN syscall: %v", err)
+			}
+			defer unix.Close(int(pidfd))
+
+			seccompFd, _, err := unix.Syscall(unix.SYS_PIDFD_GETFD, pidfd, uintptr(sync.Fd), 0)
+			if err != 0 {
+				return fmt.Errorf("error performing SYS_PIDFD_GETFD syscall: %v", err)
+			}
+			defer unix.Close(int(seccompFd))
+
+			if p.config.Config.Seccomp.ListenerPath != "" {
+				bundle, annotations := utils.Annotations(p.config.Config.Labels)
+				seccompState := &specs.ContainerProcessState{
```

Comment:
`seccompState` could be renamed to `containerProcessState` to follow the changes in runtime-spec

ditto for function `sendSeccompState()` -> `sendContainerProcessState()` below

Reply:
Right. Done.